packages/core/src/dev/activation.ts

```diff
@@ -105,6 +107,11 @@ const menuOptions: Record<DevFunction, MenuOption> = {
         detail: 'Opens editor to all Auth Connections the extension is using.',
         executor: editSsoConnections,
     },
+    forceIdeCrash: {
+        label: 'Crash: Force IDE ExtHost Crash',
+        detail: `Will SIGKILL ExtHost, { pid: ${process.pid}, sessionId: '${getSessionId().slice(0, 8)}-...' }, but the IDE itself will not crash.`,
```

Comment:
is this different than vscode's `Restart Extension Host` command? 
<img width="359" alt="image" src="https://github.com/user-attachments/assets/dfd1b70d-8e8b-426f-8faf-e08085124fb6">

Reply:
`Restart Extension Host` still results in a graceful shutdown of the extension host (`deactivate()` runs), while this one is non-graceful